<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_audit_page</class>
 <widget class="QWidget" name="new_audit_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>663</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>753</width>
    <height>663</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>753</width>
    <height>663</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>DM Mono</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(61, 64, 91);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="vspacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="instruction_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>750</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>DM Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>CHOOSE TYPE OF CIS BENCHMARK YOU WANT TO RUN</string>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextInteractionFlag::NoTextInteraction</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="options_hlayout" stretch="0,0,0,0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="hspacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="machine_type_box">
       <property name="minimumSize">
        <size>
         <width>178</width>
         <height>107</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>178</width>
         <height>107</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DM Mono Medium</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="title">
        <string>MACHINE TYPE</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="machine_type_vlayout">
          <item>
           <widget class="QRadioButton" name="workstation_btn">
            <property name="font">
             <font>
              <family>DM Mono</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>WOR&amp;KSTATION</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="server_btn">
            <property name="font">
             <font>
              <family>DM Mono</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>SE&amp;RVER</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="hspacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="benchmark_level_box">
       <property name="minimumSize">
        <size>
         <width>178</width>
         <height>107</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>178</width>
         <height>107</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DM Mono</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="title">
        <string>BENCHMARK LEVEL</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="benchmark_level_vlayout">
          <item>
           <widget class="QRadioButton" name="level1_btn">
            <property name="font">
             <font>
              <family>DM Mono</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>&amp;LEVEL 1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="level2_btn">
            <property name="font">
             <font>
              <family>DM Mono</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>LEVEL &amp;2</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="vspacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="bitlocker_btn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>BITLOCKER(WINDOWS ONLY)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="vspacer_1">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bar_bottom" stretch="0,0,0">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="cis_benchmark_btn">
       <property name="font">
        <font>
         <family>DM Mono Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>CIS benchmark pdf in browser</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(61, 64, 91);</string>
       </property>
       <property name="text">
        <string>CIS BENCHMARKS</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hspacer_1">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="continue_btn">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DM Mono Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string>background-color: rgb(255, 255, 255);
color: rgb(61, 64, 91);</string>
       </property>
       <property name="text">
        <string>CONTINUE</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
